src/components/molecules/MultiStepForm/ProgressBar/ProgressBar.tsx

```diff
@@ -1,16 +1,22 @@
 import React from 'react';
 
+import { useDispatch } from 'react-redux';
 import { IProgressBar } from './ProgressBar.model';
 import {
   StyledProgressBar,
   StyledProgressBarItem,
   StyledProgressBarItemWrapper,
 } from './ProgressBar.styled';
+import reducer from '../../../../redux/reducers/multiStepExample';
```

Comment:
We are using absolute paths. Could you change this import into 
```suggestion
import reducer from 'redux/reducers/multiStepExample';
```